tests/test_binary.nim

```diff
@@ -94,6 +94,141 @@ proc testsForSync(binaryPath: string) =
         [warn] some exercises are missing test cases
       """.dedent(8) # Not `unindent`. We want to preserve the indentation of the list items.
 
+    test "a single-exercise `sync` without `--update` exits with 1 and prints the expected output":
```

Comment:
This might be slightly much, but maybe be slightly more specific about the test case?

```suggestion
    test "a single-exercise `sync` without `--update` and missing test case exits with 1 and prints the expected output":
```

Reply:
Good catch, thanks. The previous description implied that `sync` without `--update` always exits with 1.

I've pushed https://github.com/exercism/configlet/pull/406/commits/735138dc8b4 to this PR, which tries to clarify the other test names too.

#### Before that commit:
```
[Suite] sync
  [OK] a `sync` without `--update` exits with 1 and prints the expected output
  [OK] a single-exercise `sync` without `--update` exits with 1 and prints the expected output
  [OK] when passing multiple exercises, only the final exercise is acted upon
  [OK] single-exercise `sync --update --mode=include` exits with 0 and includes the expected test cases
  [OK] `git diff` shows the expected diff
  [OK] single-exercise `sync --update --mode=exclude` exits with 0 and excludes the expected test cases
  [OK] `git diff` shows the expected diff
  [OK] single-exercise `sync --update` with input of 'y' includes as expected
  [OK] `git diff` shows the expected diff
  [OK] single-exercise `sync --update` with input of 'n' excludes as expected
  [OK] `git diff` shows the expected diff
  [OK] single-exercise `sync --update` with input of 's' exits with 1
  [OK] `git diff` shows the expected diff
  [OK] `sync --update --mode=include` exits with 0 and includes the expected test cases
  [OK] `git diff` shows the expected diff
  [OK] after syncing, another `sync --update --mode=include` performs no changes
  [OK] after syncing, a `sync` without `--update` shows that exercises are up to date
  [OK] the `git diff` output is still the same
```

#### With that commit:
```
[Suite] sync, without --update
  [OK] multiple exercises with missing test cases: prints the expected output, and exits with 1
  [OK] a given exercise with a missing test case: prints the expected output, and exits with 1
  [OK] when passing multiple exercises, only the final exercise is acted upon

[Suite] sync, with --update
  [OK] -mi: includes a missing test case for a given exercise, and exits with 0
  [OK] the diff is as expected
  [OK] -me: excludes a missing test case for a given exercise, and exits with 0
  [OK] the diff is as expected
  [OK] -mc: includes a missing test case for a given exercise when the input is 'y', and exits with 0
  [OK] the diff is as expected
  [OK] -mc: excludes a missing test case for a given exercise when the input is 'n', and exits with 0
  [OK] the diff is as expected
  [OK] -mc: neither includes nor excludes a missing test case for a given exercise when the input is 's', and exits with 1
  [OK] the diff is as expected
  [OK] -mi: includes every missing test case when not specifying an exercise, and exits with 0
  [OK] the diff is as expected
  [OK] after updating, another update using -mi performs no changes, and exits with 0
  [OK] after updating, a `sync` without `--update` shows that exercises are up to date, and exits with 0
  [OK] the diff is still the same
```